docs/semgrep-supply-chain/overview.md

```diff
@@ -50,27 +50,28 @@ _Figure 2_. Relationship between a Supply Chain rule, lockfile, CVE record, and
-containing a vulnerability, it generates a **finding**. If Semgrep Supply Chain
-also determines that there's a code pattern in the codebase that matches the
-vulnerability definition, then the finding is flagged as **reachable**
-(otherwise, it is **unreachable**).
-
-If Semgrep Supply Chain detects the use of a vulnerable version of a dependency,
-but it doesn't have a reachability rule, then the finding is **undetermined**
-(this is most common with vulnerabilities discovered prior to May 2022 or vulnerabilities with lower
-severity levels).
-
-Undetermined findings are generated by Semgrep Supply Chain's
-**[lockfile-only](/semgrep-supply-chain/glossary/#lockfile-only-rules)** rules,
-which are rules that do not perform reachability analysis. These findings are
-generated by checking the dependency's version listed in the lockfile or
-manifest against a list of versions with known vulnerabilities. These rules are
-on par with [GitHub Dependabot](https://github.com/dependabot).
-
-Specific dependency and code match findings are called **usages**, which are
-grouped by vulnerability. For each vulnerability, SCP also displays a CVE number
-corresponding to the [CVE program record](https://www.cve.org/About/Overview).
+Semgrep Supply Chains generates a **finding** any time it determines that your
+codebase leverages a package containing a vulnerability. In addition, Semgrep
+Supply Chain offers two levels of support for reachability analysis, [depending
+on your language](/supported-languages/#maturity-levels-1):
+
+* **GA**: Semgrep writes rules for all critical and high CVE
+severity levels for GA languages. That means Semgrep Supply Chain can flag all 
+your critical/high-severity findings as either reachable or unreachable.
+  * If there's a code pattern in the codebase that matches the vulnerability
+    definition, the finding is flagged as **reachable**.
+  * If you don't use the vulnerable piece of code of the library or package
+  imported, the finding is flagged as **unreachable**.
+  * If Semgrep Supply Chain determines that you use a vulnerable version of a
+  dependency, but Semgrep Supply Chain doesn't have a relevant reachability rule, it flags the finding as **undetermined**; this is most common with
+  vulnerabilities discovered before May 2022 or vulnerabilities with lower
+  severity levels.
+
+* **lockfile-only languages**: For **[lockfile-only](/semgrep-supply-chain/glossary/#lockfile-only-rules)** languages, Semgrep Supply Chain's performance is comparable to that of [GitHub's Dependabot](https://github.com/dependabot). Semgrep Supply Chain generates these findings by checking the dependency's version listed in your lockfile or manifest against a list of versions with known vulnerabilities, but it does not run reachability analysis. Because Semgrep Supply Chain doesn't run reachability analysis, it can't determine whether the vulnerability is reachable. Such vulnerabilities are, therefore, flagged as undetermined.
```

Comment:
```suggestion
* **lockfile-only languages**: For **[lockfile-only](/semgrep-supply-chain/glossary/#lockfile-only-rules)** languages, Semgrep Supply Chain's performance is comparable to that of [GitHub's Dependabot](https://github.com/dependabot). Semgrep Supply Chain generates these findings by checking the dependency's version listed in your lockfile or manifest against a list of versions with known vulnerabilities, but it does not run reachability analysis. Because Semgrep Supply Chain doesn't run reachability analysis, it can't determine whether the vulnerability is reachable. Such vulnerabilities are, therefore, flagged as **undetermined**.
```